### PR TITLE
Start a new Thread by replying to a message

### PR DESCRIPTION
**Pull request by ismailgulek**
Part of vector-im/element-ios#5068

### CODE REVIEW

File: MatrixKit/Models/Room/MXKRoomBubbleComponent.m [hunk 1]
```diff
@@ -16,9 +16,18 @@
 
 #import "MXKRoomBubbleComponent.h"
 
+#import <MatrixSDK/MatrixSDK.h>
+#import <MatrixSDK/MatrixSDK-Swift.h>
```
Comment (pixlwave) [inline MatrixKit/Models/Room/MXKRoomBubbleComponent.m:20]: I haven't yet internalised what the do's/don'ts are re importing Swift now. Is this the place you mentioned during a standup where you had to do it this way?

File: MatrixKit/Models/Room/MXKRoomDataSource.h [hunk 2]
```diff
@@ -678,6 +692,20 @@ extern NSString *const kMXKRoomDataSourceTimelineErrorErrorKey;
  */
 + (dispatch_queue_t)processingQueue;
 
+/**
+ Decides an event should be considered for asynchronous event processing.
```
Comment (pixlwave) [inline MatrixKit/Models/Room/MXKRoomDataSource.h:696]:
```suggestion
 Decides whether an event should be considered for asynchronous event processing.
```

File: MatrixKit/Models/Room/MXKRoomDataSource.h [hunk 3]
```diff
@@ -678,6 +692,20 @@ extern NSString *const kMXKRoomDataSourceTimelineErrorErrorKey;
...
+ Default implementation checks for `filterMessagesithURL` and undecryptable events sent before the user joined.
```
Comment (pixlwave) [inline MatrixKit/Models/Room/MXKRoomDataSource.h:697]:
```suggestion
 Default implementation checks for `filterMessagesWithURL` and undecryptable events sent before the user joined.
```

File: MatrixKit/Models/Room/MXKRoomBubbleComponent.m (hunk 1 above)
Comment (ismailgulek) [reply MatrixKit/Models/Room/MXKRoomBubbleComponent.m:20]: Yes, i'm waiting for MatrixKit removal right now, will update this accordingly.

Comment (pixlwave) [reply MatrixKit/Models/Room/MXKRoomBubbleComponent.m:20]: In which case LGTM 👍

**Review by pixlwave — APPROVED** [review]

**Comment by ismailgulek** [comment]
Moved to vector-im/element-ios/pull/5089 with https://github.com/vector-im/element-ios/pull/5089/commits/d7b18bb223eca1df9033a4c2f2ab93b5cd967d4c

**Pull request closed without merging**